runtime/Dockerfile

```diff
@@ -1,13 +1,12 @@
-FROM denismakogon/gocv-alpine:3.4.2-buildstage as build-stage
+FROM denismakogon/gocv-alpine:4.0.0-buildstage as build-stage
 FROM denismakogon/ffmpeg-alpine:4.0
 
 LABEL maintainer="Denis Makogon. mail: lildee1991@gmail.com"
 
-# OpenCV 3.4.1 shared objects from build-stage
+# OpenCV 4.0.0 shared objects from build-stage
 COPY --from=build-stage /usr/local/lib64 /usr/local/lib64
-COPY --from=build-stage /usr/local/lib64/pkgconfig/opencv.pc /usr/local/lib64/pkgconfig/opencv.pc
-COPY --from=build-stage /usr/local/include/opencv /usr/local/include/opencv
-COPY --from=build-stage /usr/local/include/opencv2 /usr/local/include/opencv2
+COPY --from=build-stage /usr/local/lib64/pkgconfig/opencv4.pc /usr/local/lib64/pkgconfig/opencv4.pc
+COPY --from=build-stage /usr/local/include/opencv4/opencv2 /usr/local/include/opencv4/opencv2
```

Comment:
i found that i also needed to update some of the steps to get glibc down on lines 30-32

```
 wget -q -O /etc/apk/keys/sgerrand.rsa.pub https://alpine-pkgs.sgerrand.com/sgerrand.rsa.pub && \
 wget https://github.com/sgerrand/alpine-pkg-glibc/releases/download/2.28-r0/glibc-2.28-r0.apk && \
 apk add --no-cache  glibc-2.28-r0.apk && \
```